src/rp2_common/hardware_dma/include/hardware/dma.h

```diff
@@ -761,6 +765,34 @@ inline static void dma_sniffer_set_byte_swap_enabled(bool swap) {
         hw_clear_bits(&dma_hw->sniff_ctrl, DMA_SNIFF_CTRL_BSWAP_BITS);
 }
 
+/*! \brief Enable the Sniffer output invert function
+ *  \ingroup hardware_dma
+ *
+ * Perform a bit inversion on the sniffed data output
+ *
+ * \param invert Set true to enable output inversion
+ */
+inline static void dma_sniffer_set_output_invert_enabled(bool invert) {
+    if (invert)
+        hw_set_bits(&dma_hw->sniff_ctrl, DMA_SNIFF_CTRL_OUT_INV_BITS);
+    else
+        hw_clear_bits(&dma_hw->sniff_ctrl, DMA_SNIFF_CTRL_OUT_INV_BITS);
+}
+
+/*! \brief Enable the Sniffer output bit reversal function
+ *  \ingroup hardware_dma
+ *
+ * Perform a bit inversion on the sniffed data output
```

Comment:
same comment as above, and also this is bit reverse not bit inversion

Reply:
Thanks, agreed, resolved (I hope).